### ISSUE TITLE
External link text

### ISSUE THREAD

**Issue by Amantel** [spencermountain/wtf_wikipedia#69]
Hello! 

While parsing sentence we keep nice array of links (both external and internal), but sometimes there is relatively hard to map them to actual sentence.

Example:
First section of https://en.wikivoyage.org/w/index.php?title=Barcelona/Eixample&action=edit
We have 
```
The '''[http://w110.bcn.cat/portal/site/Eixample Eixample]''' is the quarter designed during the middle of the 19th century by Ildefons Cerdà, expanding the medieval city of [[Barcelona]] into space left empty for defense outside the city walls.
```
Which translates to: 
```
text:"The ''' Eixample''' is the quarter designed during the middle of the 19th century by Ildefons Cerdà, expanding the medieval city of Barcelona into space left empty for defense outside the city walls."
```
```
links: Array(2) [Object, Object]
0:Object {type: "external", site: "http://w110.bcn.cat/portal/site/Eixample", text: ""}
1:Object {page: "Barcelona", text: "Barcelona"}
```
When I will try to print this sentence should I:
1. Use some regexp to find first (second/third) of `''' '''` templates an insert first link there, if type is external? 
2. Find regexp of text if type is page (or other) and insert link to wikipage with text (Barcelona)?

**Comment by spencermountain** [comment]
hey, nah that just looks like a bug. It should definitely have `text: 'Eixample'`, not sure why it hasn't.
will add it to the list.

**Comment by spencermountain** [comment]
fixed in 2.2.0
